kernel/procfs.c

```diff
@@ -97,6 +101,25 @@ int procfs_pid_setup(pid_t pid)
         ECHECK(myst_mkdirhier(locals->tmp_path, 777));
     }
 
+    /* Create /proc/sys/kernel directory */
```

Comment:
/proc/sys/kernel is not a pid specific entry. `procfs_pid_setup` is called on process creation.

Reply:
Recommend moving this to `create_proc_root_entries`.